helpers/load_test_interface.go

```diff
@@ -290,7 +290,8 @@ func NighthawkLoadTest(opts *models.LoadTestOptions) (map[string]interface{}, *p
 		OpenLoop:                             &wrappers.BoolValue{Value: false},
 		JitterUniform:                        durationpb.New(0 * time.Second),
 		ExperimentalH2UseMultipleConnections: &wrappers.BoolValue{Value: false},
-		Labels:                               make([]string, 0),
+		// Labels:                               make([]string, 0),
+		Labels:                               labels,
```

Comment:
The golang lint test is failing here because you're using a using string in place of a struct literal, could you please take a look.

Reply:
@leecalcote @warunicorn19 Thanks for pointing it out. I've made the appropriate fix locally but haven't committed yet. I think I'm close to a potential solution